redhat-cop/pelorus
# Fallback to BuildConfig when Build does not have git uri

## Describe the behavior changes introduced in this PR
This change allows fallback behavior in the commit time exporter to look for the Git URI in the BuildConfig object if the Build object does not specify this information.

In the case it is not found in either location, then there is no change in behavior.

## Linked Issues?

resolves #222 

## Testing Instructions

Although this is easily reproducible in my customer's environment with a Docker strategy BuildConfig, I'm not sure how it happens that the spec.source.git information is not preserved from the BuildConfig to the Build object.  See the reported issue this resolves for an example.

@redhat-cop/mdt
